### ISSUE TITLE
Character Traits Excess Information

### ISSUE THREAD

**Issue by shadowfray** [Bathtor/EPSheet#76]
The character sheet area for traits has some extra lines which seem like they're not supposed to be there. [Here is a screenshot](https://gyazo.com/576fa74f6f0b836be51ab1a00523532a). As well, the active skills seem to suffer a [similar problem](https://gyazo.com/c8241e2f64cef28a266a860f71a9aa67). If these are intentional, could there be a way to hide them to make it more legible?

**Comment by Bathtor** [comment]
Crap, looks like something about the way Roll20 handles internationalisation changed. It's showing the i18n key for the text instead of the localised value.

I'll try to find some time to fix it next weekend.

**Comment by Bathtor** [comment]
Ok, I've spent the day looking into this (and a bunch of other new issues with the sheet stemming from dependency updates).

I've reproduced it in a fairly minimal sheet, at it seems that the Roll20 Sheet feature that allows lookups of translation keys based on a field value is currently broken in general.
It was reported for the Shaped sheet as well here: https://app.roll20.net/forum/post/10510203/shaped-character-sheet-issue
And I personally checked Roll20's official D&D 5E sheet and it seems broken there, too. Just the translation key happens to be basically the same as the value in English there, so it's not so bothersome.

I'll give Roll20 a week to reply and let me know if/when they are planning to fix this.
If I don't get a helpful answer by then, I think it can be worked around by using sheet workers, so I may explore that option next weekend instead.
